stories/UploadFiles.stories.tsx

```diff
@@ -2,6 +2,7 @@ import { Meta, Story } from '@storybook/react'
 import { UPLOAD_ADAPTER, UploadAdapter, UpupUploader } from '../src'
 import useUpup from '../src/hooks/useUpup'
 import { UpupUploaderProps } from '../src/UpupUploader'
+import React from 'react'
```

Comment:
You can remove this import, already configured babel present to accept jsx without React import

Reply:
clearly that didn't work otherwise i wouldn't put it

Reply:
@MohamedBechirMejri  no problem, we will fix it